framework/guid/model.py

```diff
@@ -20,6 +20,26 @@ class Guid(StoredObject):
     _id = fields.StringField(primary=True)
     referent = fields.AbstractForeignField()
 
+    @classmethod
+    def generate(self, referent=None):
+        while True:
+            # Create GUID
+            guid_id = ''.join(random.sample(ALPHABET, 5))
```

Comment:
@sloria I have not, I personally think that's a bit out of scope of this PR. Nothing has changed about the amount of GUIDs being generated nor how they are being generated.
That being said I'd be happy to address it if you'd like me too.